resources/feResource.cc

```diff
@@ -163,8 +163,17 @@ void feInitResources(const char* argv0)
   {
     //WarnS("illegal argv[0]==NULL");
     feArgv0 = (char*)malloc(MAXPATHLEN+strlen("/Singular"));
-    getcwd(feArgv0, MAXPATHLEN);
-    strcat(feArgv0,"/Singular");
+    if (feArgv0 != NULL)
```

Comment:
if feArgv0 is NULL, we are out of memeory in a very early stage of the program (and argv[0] is NULL, very unusal)
it is better to abort with a meaningful message (out of memory in initialitasation for example)